tests/integration/test.bulk_docs.js

```diff
@@ -320,6 +320,89 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    it('Deleting _local docs with bulkDocs' , function () {
+      var db = new PouchDB(dbs.name);
+
+      var rev1;
+      var rev2;
+      var rev3;
+      return db.put({_id: '_local/godzilla'}).then(function (info) {
+        rev1 = info.rev;
+        return db.put({_id: 'mothra'});
+      }).then(function (info) {
+        rev2 = info.rev;
+        return db.put({_id: 'rodan'});
+      }).then(function (info) {
+        rev3 = info.rev;
+        return db.bulkDocs([
+          {_id: 'mothra', _rev: rev2, _deleted: true},
+          {_id: '_local/godzilla', _rev: rev1, _deleted: true},
+          {_id: 'rodan', _rev: rev3, _deleted: true}
+        ]);
+      }).then(function () {
+        return db.allDocs();
+      }).then(function (info) {
+        info.rows.should.have.length(0);
+        return db.get('_local/godzilla').then(function () {
+          throw new Error('expected 404');
+        }, function (err) {
+          should.exist(err);
+        });
+      });
+    });
```

Comment:
^ @daleharvey This test passes on CouchDB 1.6, so yes, it lets you delete a _local doc within _bulk_docs.